tests/component/copypaste-external.spec.tsx

```diff
@@ -14,13 +14,16 @@ test.beforeEach(async ({}, testInfo) => {
   testInfo.snapshotSuffix = "";
 });
 
+// TODO: These tests should not be used for now. While they do pass, the
```

Comment:
wdym with "should not be used"?

Reply:
The functionality they test doesn't work yet, so the tests and snapshots aren't useful atm